File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/StickerFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.sticker.Sticker;

public class StickerFilterValidator extends FilterValidator<Sticker> {

    public StickerFilterValidator(CustomValidator<Sticker> validator) {
        super(Sticker.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/StickerFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.std.sticker.Sticker
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class StickerFilterValidator(validator: Validator<Sticker>) : FilterValidator<Sticker>(Sticker::class, validator)
